src/core/getProfits.ts

```diff
@@ -103,7 +128,9 @@ export async function getProfits({
       const profit =
         endPositionValue + withdrawal - deposit - startPositionValue
 
-      if (adapter.getProtocolDetails().positionType == PositionType.Borrow) {
+      const isBorrow =
+        adapter.getProtocolDetails().positionType == PositionType.Borrow
+      if (isBorrow) {
         profit * -1
```

Comment:
Weird that we didn't caught this before but... this expression is not being assigned anywhere.